x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/multi_page_layout/components/page_steps/install_agent/install_agent_standalone.tsx

```diff
@@ -79,6 +83,17 @@ export const InstallElasticAgentStandalonePageStep: React.FC<InstallAgentPagePro
     setYaml(fullAgentPolicyToYaml(fullAgentPolicy, safeDump));
   }, [fullAgentPolicy]);
 
+  const onCreateApiKey = useCallback(async () => {
+    const res = await sendCreateStandaloneAgentAPIKey({
+      name: crypto.randomBytes(16).toString('hex'),
+    });
+    if (res.error) {
+      throw res.error;
```

Comment:
We should probably do something with that error, I tried to simulate an error and nothing is shown to the user. 

Usually we use a toast for that like this https://github.com/jillguyonnet/kibana/blob/992ab38c41942ddfbd1e539296aeafc8ae1d3bb2/x-pack/plugins/fleet/public/applications/fleet/hooks/use_service_token.ts#L27-L31

Reply:
Good catch! That slipped my radar.

Reply:
@nchaulet I'm taking over this PR from Jill while she's away. I pushed https://github.com/elastic/kibana/pull/187133/commits/dc25e8265b45d9f430f0170328151dba1f164e28 where I added the toast notifications and also moved this callback to a hook, since it was also used in `fleet/public/components/agent_enrollment_flyout/steps/compute_steps.tsx`